client/templates/employees.js

```diff
@@ -21,6 +21,28 @@ Template.employees.helpers({
     }
 });
 
+Template.employeeInput.helpers({
+    checkDay : function(day){
+        if(this){
```

Comment:
Maybe i don´t really understand the problem, but would it not be fixed by changing this line to `if(this.hasOwnProperty("freeDays")){`?